pkgs/tools/admin/pulumi/update.sh

```diff
@@ -1,15 +1,19 @@
 #!/usr/bin/env bash
 
-VERSION="2.2.1"
+VERSION="2.6.1"
 
-declare -A plugins
-plugins=(
-    ["aws"]="2.5.0"
-    ["gcp"]="3.6.0"
-    ["random"]="2.1.1"
-    ["kubernetes"]="2.2.0"
-    ["postgresql"]="2.1.1"
-)
+# bash 3 compatible for Darwin
+plugins=(       "aws"    "gcp"    "random" "kubernetes" "postgresql");
```

Comment:
Is there any alternative way to format plugins and versions, as these means you must count which plugin belongs to which version and it's easy to make mistake? This will be especially problematic when we will have more plugins. Maybe we can format `<plugin_name>:version`?

Reply:
yes, I have a better solution as you have suggested